[Docs] add set command examplee to sql api (#18027)
apache/druid: docs/querying/using-caching.md

  }
}
```

In this example the user has set `populateCache` to `false` to avoid filling the result cache with results for segments that are over a year old. For more information, see [Druid SQL client APIs](../api-reference/sql-api.md).

> You can also use the SET command to specify cache options directly within your SQL query string. For details about SET, see [SET statements](../querying/sql.md#set). 

Reviewer: ```suggestion
You can also use the SET command to specify cache options directly within your SQL query string. For more information, see [SET statements](../querying/sql.md#set-statements). 
```

  }
}
```

In this example the user has set `populateCache` to `false` to avoid filling the result cache with results for segments that are over a year old. For more information, see [Druid SQL client APIs](../api-reference/sql-api.md).

You can also use the SET command to specify cache options directly within your SQL query string. For example: 

```
{
  "query" : "SET useCache=true; SET populateCache=false; SELECT COUNT(*) FROM data_source WHERE foo = 'bar' AND __time > TIMESTAMP '2020-01-01 00:00:00'"
}
```

For details about SET, see [SET statements](../querying/sql.md#set-statements). 

> 

Reviewer: Let's actually not add a duplicate example in this doc. The other docs were more introductory, so it made sense to show multiple ways. Instead, let's just do a short one-liner letting the user know that there's another route to set the parameter. "You can also set the context parameter directly in `query` using the SET command. For details, see <link to the sql-api example>."

Author: resolved in the commit below.
